code/modules/1713/jobs/russian.dm

```diff
@@ -1731,3 +1731,294 @@
 	H.setStat("bows", STAT_NORMAL)
 	H.setStat("medical", STAT_MEDIUM_LOW)
 	return TRUE
+
+// Modern
+/datum/job/russian/modern_lieutenant
+	title = "Poruchik"
+	en_meaning = "Lieutenant"
+	rank_abbreviation = "Lt."
+	spawn_location = "JoinLateRUCap"
+
+	is_operation_falcon = TRUE
+	is_modernday = TRUE
+	is_ww2 = FALSE
+	is_commander = TRUE
+	is_officer = TRUE
+
+	uses_squads = TRUE
+	whitelisted = TRUE
+
+	additional_languages = list("English" = 70)
+	min_positions = 1
+	max_positions = 1
+
+/datum/job/russian/modern_lieutenant/equip(var/mob/living/human/H)
+	if (!H)	return FALSE
+//shoes
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots/jackboots2(H), slot_shoes)
+//clothes
+	H.equip_to_slot_or_del(new /obj/item/clothing/under/russian(H), slot_w_uniform)
+//armor
+	var/obj/item/clothing/under/uniform = H.w_uniform
+	var/obj/item/clothing/accessory/armor/coldwar/pasgt/pasgt_armor = new /obj/item/clothing/accessory/armor/coldwar/pasgt(null)
```

Comment:
PASGT for modern russians?
Put either the 6b5 or the generic plate carrier.
Here's the link to the old russoukraine jobs we had:
https://github.com/Civ13/Civ13/blob/37c1b0d5ace50f7ad6e09f5d9a3e896155e8b33c/code/modules/1713/jobs/russoukrainewar.dm